ZipBuilder/ZipBuilderV2Plan.md

```diff
@@ -0,0 +1,58 @@
+V2 will add an option to generate a zip distribution of binary frameworks from an arbitrary list
```

Comment:
There are a few different capitalizations throughout (and throughout existing docs, which should be changed eventually). In an effort to keep it consistent, what do you think about:

- `zip` referring to a file that has the `.zip` extension, `Zip` if at the start of a sentence
- the `Zip Builder` (capitalized with a space) for the *product* that will be building `zip` files
- `ZipBuilder` for referring to the code and tool itself

I'm open to other suggestions as well but we should try to keep it consistent throughout if possible.

Reply:
Done.